backend/database/structure.sql

```diff
@@ -267,20 +267,50 @@ COLLATE = ascii_general_ci;
 -- -----------------------------------------------------
 -- Table `unipept`.`ec_cross_references`
 -- -----------------------------------------------------
-CREATE  TABLE IF NOT EXISTS `unipept`.`ec_cross_references` (
-  `id` INT UNSIGNED NOT NULL AUTO_INCREMENT ,
-  `uniprot_entry_id` INT UNSIGNED NOT NULL ,
-  `ec_id` VARCHAR(12) NOT NULL ,
-  PRIMARY KEY (`id`) ,
-  INDEX `fk_ec_reference_uniprot_entries` (`uniprot_entry_id` ASC) ,
+#CREATE  TABLE IF NOT EXISTS `unipept`.`ec_cross_references` (
```

Comment:
why are these commented?


_[Original comment](https://github.ugent.be/unipept/unipept/pull/528#discussion_r3368) by @bmesuere on Mon Oct 19 2015 at 14:43._